src/senders/report_stream_sender.go

```diff
@@ -30,11 +30,13 @@ func NewSender() (Sender, error) {
 		slog.Error("Unable to initialize credential getter", slog.Any(utils.ErrorKey, err))
 		return Sender{}, err
 	}
+	// ca-phl-reportstream-private-key
+	reportStreamPrivateKeyName := utils.CA_PHL + "-reportstream-private-key-" + utils.EnvironmentName() // pragma: allowlist secret
 
 	return Sender{
 		baseUrl:          os.Getenv("REPORT_STREAM_URL_PREFIX"),
-		privateKeyName:   os.Getenv("FLEXION_PRIVATE_KEY_NAME"),
-		clientName:       os.Getenv("FLEXION_CLIENT_NAME"),
+		privateKeyName:   reportStreamPrivateKeyName,
+		clientName:       os.Getenv("CA_PHL_CLIENT_NAME"),
 		credentialGetter: credentialGetter,
 	}, nil
 }
```

Comment:
The `scope` on line 80 below needs to be updated too. It should probably be in a constant somewhere

Reply:
Ah yes.  We'll update that when we get the `ca-phl` credentials figured out.  I didn't want this PR to depend on getting those credentials figured out.

Reply:
(discussed in person - we should make these updates be ACs for the prod setup card so we don't risk losing track of them. The code will still work without these updates, and would send data with incorrect ownership/attribution)